org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsd/participants/XSDErrorCode.java

```diff
@@ -163,7 +164,18 @@ public static Range toLSPRange(XMLLocator location, XSDErrorCode code, Object[]
 		}
 		case EmptyTargetNamespace :
 			return XMLPositionUtility.selectAttributeValueAt(XSDUtils.TARGET_NAMESPACE_ATTR, offset, document);
+		case src_import_3_1: { // src-import.3.1: The namespace attribute, _, of an <import> element information item must be identical to the targetNamespace attribute, _, of the imported document.

```

Comment:
Would it be easier to reference something like http://svn.apache.org/viewvc/xerces/java/trunk/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties?view=log as opposed to copying the full error message ?

Is there a way to reference that file ?

Reply:
I added the comment just so we know what error is thrown, but I guess it's inconsistent with the other cases, so I can remove it.

However, I feel it is helpful to have somewhere that lists the message(s) thrown so that (in the future) a developer knows where to look specifically when handling this error.

Reply:
I would maybe  add http://svn.apache.org/viewvc/xerces/java/trunk/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties?revision=1869198&view to the top of the file. I think that's the upstream resource.